tools/cldr-unittest/src/org/unicode/cldr/unittest/TestPaths.java

```diff
@@ -46,11 +45,10 @@ public static void main(String[] args) {
     }
 
     public void VerifyEnglishVsRoot() {
-        Set<String> rootPaths = CollectionUtilities.addAll(testInfo
-            .getRoot().iterator(),
-            new HashSet<String>());
-        Set<String> englishPaths = CollectionUtilities.addAll(testInfo
-            .getEnglish().iterator(), new HashSet<String>());
+        HashSet<String> rootPaths = new HashSet<String>();
```

Comment:
Cases like this have to be unpacked so you make the container and then add to it, rather than passing it into the utility method.